### PR TITLE
CSV import createEntities

### PR DESCRIPTION
**Pull request by aufi**
Updating CSV import adding an optional field "createEntities" (true by
default) to create missing BusinessServices,TagTypes and Tags in the Hub database.

https://issues.redhat.com/browse/TACKLE-691
https://issues.redhat.com/browse/TACKLE-693

### CODE REVIEW

**Comment by aufi** [comment]
Can be tested (and works for me locally) with:
```
curl -F "createEntities=false" -F "file=@sample_import_file.csv" localhost:8080/importsummaries/upload
```

File: importer/manager.go [hunk 1]
```diff
@@ -40,7 +42,7 @@ func (m *Manager) Run(ctx context.Context) {
 // unprocessed imports.
 func (m *Manager) processImports() (err error) {
 	list := []model.Import{}
-	db := m.DB.Preload("ImportTags")
+	db := m.DB.Preload("ImportTags").Preload("ImportSummary")
```
Comment (aufi) [inline importer/manager.go:45]: The createEntities field is on ImportSummary, need to retrieve it with Import.

File: importer/manager.go [hunk 2]
```diff
@@ -143,24 +145,90 @@ func (m *Manager) createApplication(imp *model.Import) (ok bool) {
 		}
 	}
 
+	// Assign Business Service
 	businessService := &model.BusinessService{}
 	result := m.DB.Select("id").Where("name LIKE ?", imp.BusinessService).First(businessService)
```
[the remark this answers is not in the document]
Comment (aufi) [reply importer/manager.go]: Good point, I don't see any relevant reason for that - updated.

**Comment by ibolton336** [comment]
Tested and looks good. Works with https://github.com/konveyor/tackle2-ui/pull/317

**Review by ibolton336 — APPROVED** [review]

**Pull request merged**